edx_proctoring/api.py

```diff
@@ -652,7 +654,7 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
                 cascade_effects=False
             )
 
-    if to_status == ProctoredExamStudentAttemptStatus.submitted:
+    if to_status == ProctoredExamStudentAttemptStatus.waiting_for_app_timeout:
```

Comment:
sorry to suggest this while you are so far through, but the term "timeout" is a bit negative. Can we use waiting_for_app_shutdown instead? Hopefully, it's just a search & replace operation.